packages/core/contracts/Escrow.sol

```diff
@@ -228,63 +223,46 @@ contract Escrow is IEscrow, ReentrancyGuard {
         require(aggregatedBulkAmount < BULK_MAX_VALUE, 'Bulk value too high');
 
         if (balance < aggregatedBulkAmount) {
-            return bulkPaid;
+            return false;
```

Comment:
use a require instead, otherwise we would be paying for a transaction that does nothing